code/main/edge/edgeRequestWorker.brs

```diff
@@ -224,6 +246,65 @@ function _adb_EdgeRequestWorker() as object
             return identityMap
         end function,
 
+        _processResponseOnSuccess: function(edgeResponse as object) as void
+            try
+                responseString = edgeResponse.getResponseString()
+                responseJson = ParseJson(responseString)
+                if _adb_isEmptyOrInvalidMap(responseJson)
+                    _adb_logError("EdgeRequestWorker::_processResponseOnSuccess() - Failed to parse response: (" + FormatJson(responseString) + ")")
+                    return
+                end if
+
+                handles = responseJson.handle
+                if _adb_isEmptyOrInvalidArray(handles)
+                    _adb_logVerbose("EdgeRequestWorker::_processResponseOnSuccess() - Empty handles in the response.")
+                    return
+                end if
+
+                for each handle in handles
+                    if _adb_isEmptyOrInvalidMap(handle)
+                        continue for
+                    end if
+
+                    if handle.type = "locationHint:result"
+                        m._extractAndCacheLocationHint(handle)
+                    else if handle.type = "state:store"
+                        m._extractAndCacheStateStore(handle)
```

Comment:
Can we move this logic to the `_adb_KonductorConfig` function? It could be cleaner when adding more response handling logic in the future.

Reply:
Do you mean extractAndCacheLocationHint and extractAndCacheStateStore logic? or you want to pass the entire handle to konductorConfig. I was thinking we might have different handles which might not apply to KonductorConfig and would be helpful to keep the filtering on handle type here. Though it will not be a big difference if we do it inside KonductorConfig class as well. What are your thoughts?

Reply:
Oh, I meant to move the extractAndCacheLocationHint and extractAndCacheStateStore logic to `konductorConfig`. Yes, I was thinking the same. In the future, we may add logic to handle other types of handles in different functions.